src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -62,3 +62,4 @@ export const MAP_TYPE = {
 export const TRADITIONAL_OCL_HOST = urlConfig.TRADITIONAL_OCL_HOST;
 
 export const CUSTOM_SOURCE = 'Custom';
+export const SELECT_NAME = 'source';
```

Comment:
What does select name mean?

Reply:
It is a constant that specifies the name attribute of the select tag.

Reply:
Can you point me to this select tag and its name attribute? Do you mean that its name attribute value should be source?

Reply:
Yes, in the event a different source of a mapping is selected, the value of the name attribute is 'source'.

Reply:
Can you point me to this select tag and its name attribute?

Reply:
![Uploading Screen Shot 2019-03-29 at 09.36.43.png…]()
The select tag I meant is in the Create Mapping component.

Reply:
In that case ATTRIBUTE_NAME_SOURCE makes more sense.

Reply:
Let me change it right away.

Reply:
I have changed the constant name to ATTRIBUTE_NAME_SOURCE.